test/classes/ferguson/generic-field/generic-field-record-not-inited3.future

```diff
@@ -0,0 +1,2 @@
+bug: record with generic field default-of problem
+#9568
```

Comment:
As far as I can tell, the `...inited3.chpl` test is identical to the `...inited2.chpl` test, but has a `.future` with a different bug description, but citing the same issue number.  The `.good` files are the same as well.  Was that what was intended?

Reply:
The .future and .good contents were intentional but I meant to commit the version of the .chpl that included the commented out code. Now fixed, thanks.

Reply:
I intentionally didn't put one because it would be a pain to maintain (it includes all _defaultOf overloads in a compile-time error message)